types/phantom/index.d.ts

```diff
@@ -34,6 +34,7 @@ export interface WebPage {
     property<T>(key: string, value: T): Promise<void>;
 
     setting<T>(key: string): Promise<T>;
+    setting<T>(key: string, value: T): Promise<T>;
```

Comment:
it's better to keep them separate because without `value` it works as a getter whereas when `value` is set it's a setter